contractdata/AMBROSE/_AMBROSE_CHALLENGES.json

```diff
@@ -3090,6 +3090,123 @@
                     "Tags": ["argon-pack", "story", "live", "medium"]
                 }
             ]
+        },
+        {
```

Comment:
This challenge does not trigger a pop-up in game, but the server completes it.

Reply:
Looks like it settles when the mission finished. But somehow.

Reply:
Yeah, the game doesn't really like what you've done, not sure why, it looks fine and Peacock handles it perfectly fine. Thanks for the quick fix, I'll test it once more and then it looks good to merge!